pkg/apis/core/types.go

```diff
@@ -5606,6 +5606,7 @@ type TopologySpreadConstraint struct {
 	// MaxSkew describes the degree to which pods may be unevenly distributed.
 	// When `whenUnsatisfiable=DoNotSchedule`, it is the maximum permitted difference
 	// between the number of matching pods in the target topology and the global minimum.
+	// The global minimum indicates the minimum number of pods in an eligible domain.
 	// For example, in a 3-zone cluster, MaxSkew is set to 1, and pods with the same
 	// labelSelector spread as 1/1/0:
```

Comment:
say that, in this example, the global minimum is zero. Although it would be more useful to have the example as 2/2/1, such that global minimum is not zero, to distinguish from the `minDomains` example.